vignettes/getting_started_project.Rmd

```diff
@@ -68,14 +69,15 @@ In case of a previous set-up, it will take that into account.
 - Do you want to check spelling with `check_spelling()`.
 - Do you want to update the citation files?
 
-# To do once after activating version control
+# To do once after activating version control (#setup-github)
 
 - Make sure you have at least one commit.
 - Checkout the main branch.
 - Login to https://github.com
 - Create a new repository without adding a `README`, `.gitignore` and license.
   `checklist` will add those files for you.
-- Run the code listed under "...or push an existing repository from the command line" in the command line.
+  (A public repository for organisation INBO can be created in https://github.com/inbo on tab `Repositories`.)
+- Copy the code listed in your newly created repository on GitHub under "...or push an existing repository from the command line" and run it in the command line.
```

Comment:
```suggestion
- Copy the code listed in your newly created repository on GitHub under "...or push an existing repository from the command line" and run it in the terminal.
```

For Windows users, terminal might be a better word, as they have a terminal in RStudio while the (Windows) command line might not give the desired result.